rcl/include/rcl/remap.h

```diff
@@ -28,13 +28,15 @@ extern "C"
 {
 #endif
 
-struct rcl_remap_impl_t;
+struct rcl_remap_impl_s;
+
+typedef struct rcl_remap_impl_s rcl_remap_impl_t;
```

Comment:
Same question about removing the typedef.

Reply:
Same as above.